.travis.yml

```diff
@@ -14,3 +14,17 @@ before_script: "rubocop"
 addons:
   srcclr: true
 after_success: "coveralls"
+
+jobs:
+  include:
+    - stage: 'Integration tests'
+      env: SDK=ruby
+      language: python
+      before_install:
+        - "pip install awscli"
```

Comment:
I still think semantically, `pip install awscli` should be part of the `install` step and the cpoying of the artifacts should be in `before_script`